src/QMCWaveFunctions/Jastrow/CountingJastrow.h

```diff
@@ -376,7 +376,7 @@ class CountingJastrow : public WaveFunctionComponent
     return Jgrad[iat];
   }
 
-  PsiValueType ratioGrad(ParticleSet& P, int iat, PosType& grad_iat)
+  PsiValueType ratioGrad(ParticleSet& P, int iat, GradType& grad_iat) override
```

Comment:
I had to change `PosType` to `GradType` in for `override` to work in both real and complex builds

Reply:
Nice catch